### ISSUE TITLE
Assignment05 is ready for grading

### ISSUE THREAD

**Issue by jtanggsd** [SES-5215-F2020/jtang-quant#5]
Hi,

Both the .Rmd and .html file are ready for grading. Please view the html file here: https://ses-5215-f2020.github.io/jtang-quant/assignment05_jt. Thanks very much.

**Comment by c-voulgaris** [comment]
You've definitely met the requirements for the assignment here. The overall narrative of the assignment is kind of hard to follow because you're trying so many different things. I love to see the evidence of so much experimentation though.
